pos_stock_picking_invoice_link/models/stock_move.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    pos_order_line_id = fields.Many2one(
+        comodel_name='pos.order.line',
+        string='Related POS Order Line',
+    )
+
+    @api.model
```

Comment:
Perhaps can use '@api.model_create_multi' here

Reply:
It'd be great, but this is v11 :wink: